doc/source/serve/advanced-guides/advanced-autoscaling.md

```diff
@@ -283,4 +283,4 @@ You may observe that deployments are scaling down too quickly. Instead, you may
 
 | `downscaling_factor = 1` | `downscaling_factor = 0.5` |
 | ------------------------------------------------ | ----------------------------------------------- |
-| ![downscale-smooth-before](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/downscale_smoothing_factor_before.png) | ![downscale-smooth-after](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/downscale_smoothing_factor_after.png) |
\ No newline at end of file
+| ![downscale-smooth-before](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/downscale_smoothing_factor_before.png) | ![downscale-smooth-after](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/downscale_smoothing_factor_after.png) |
```

Comment:
@zhangyilun What's the difference between these 2 lines? I don't see any changes.

Reply:
This is due to the original file doesn't include new line at the end of the file. This PR adds that in probably from IDE